### PR TITLE
Corporate feature

### PR DESCRIPTION
**Pull request by darokel**
Places the corporate page behind a feature flag.

### CODE REVIEW

**Comment by asmega** [comment]
:+1:

File: features/corporate_page.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Feature: Settings
+  As a corporate partner
+  I want to be able to view more information on partnering
+  so that I am aware of different ways of partnering with MAS.
```
Comment (benlovell) [inline features/corporate_page.feature]: Capitalise `so` and the full stop can go to I reckon.

**Comment by darokel** [comment]
@benlovell All good?

File: features/corporate_page.feature [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+  @enable-corporate
+  Scenario: Accessing the corporate page
+    When  I visit the corporate page
+    Then  I should be informed that I am on the correct page
+
+  Scenario: Corporate page is not accessible when feature is disabled
+    When I visit the corporate page
```
Comment (benlovell) [inline features/corporate_page.feature]: If we are going to align the non-gherkin reserved stuff (which I don't necessarily agree with myself, but it seems other folks like to), then it should be consistent. This one is out a little.

File: features/support/ui/pages/corporate.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require_relative '../page'
+
+module UI::Pages
+  class Corporate < UI::Page
+    set_url '{/locale}/corporate'
+
```
Comment (benlovell) [inline features/support/ui/pages/corporate.rb]: Whoops.

File: spec/controllers/corporate_controller_spec.rb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-RSpec.describe CorporateController, type: :controller do
+RSpec.describe CorporateController, type: :controller, features: [:corporate] do
```
Comment (benlovell) [inline spec/controllers/corporate_controller_spec.rb]: I'm certain you don't need to wrap `:corporate` in an array. They get normalised in the library.

Comment (darokel) [inline spec/controllers/corporate_controller_spec.rb]: There are a couple of other files which follow the same format. I personally don't mind whether we use arrays or not (for single features) but I'd like consistency. I suppose having single features in an array clarify more intently that multiple features can be passed in. What do you reckon?

File: features/support/ui/pages/corporate.rb (hunk 3 above)
Comment (darokel) [inline features/support/ui/pages/corporate.rb]: Dang!

**Pull request merged**